Rubberduck.Parsing/Symbols/Accessibility.cs

```diff
@@ -9,4 +9,13 @@ public enum Accessibility
         Global = 5,
         Static = 6
     }
+
+    public static class AccessibilityExtensions
+    {
+        public static string CodeString(this Accessibility access)
```

Comment:
I like the `CodeString` name, but given the existence of a completely unrelated `CodeString` type, I think a `<summary>` comment can't hurt here. Thoughts?

```suggestion
        /// <summary>
        /// Gets the string/token representation of an accessibility specifier.
        /// </summary>
        /// <remarks>Implicit accessibility being unspecified, yields an empty string.</remarks>
        public static string CodeString(this Accessibility access)
```